src/components/comment-placer/CommentPlacer.js

```diff
@@ -0,0 +1,187 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { findDOMNode } from 'react-dom';
+import FocusManager from './focus-manager';
+import FadeAndGrowTransition from './fade-and-grow-transition';
+import FadeTransition from './fade-transition';
+
+export default class CommentPlacer extends Component {
+    static propTypes = {
+        animateOnMount: PropTypes.bool,
+        animateOnUnmount: PropTypes.bool,
+        scrollOnMount: PropTypes.bool,
+        autofocus: PropTypes.bool,
+        children: PropTypes.element.isRequired,
+        animation: PropTypes.oneOf(['fade', 'fade-and-grow']).isRequired,
```

Comment:
This shouldn't be required and instead should have a default value, e.g.: `fade`